responsibleai/responsibleai/rai_insights/rai_insights.py

```diff
@@ -602,7 +665,14 @@ def _get_dataset(self):
                 "Unsupported dataset type") from ex
         if dataset is not None and self.model is not None:
             try:
-                predicted_y = self.model.predict(dataset)
+                predict_dataset = dataset
+                if self._feature_metadata is not None:
```

Comment:
nit; you can collapse the two ifs to reduce nesting with an and, maybe might be good to use:
```
metadata = self._feature_metadata
meta_exists = metadata is not None
dropped_features_exist = meta_exists and metadata.dropped_features is not None
if dropped_featues_exists and len(metadata.dropped_features)  != 0:
    predict_dataset = predict_dataset.drop(
                            self._feature_metadata.dropped_features, axis=1)
```

Reply:
Updated!